Try Implementing inline-block footer links as described in #205

kadamwhite opened the pull request:
![image](https://cloud.githubusercontent.com/assets/442115/10532035/7a190954-7387-11e5-86c9-60541542160b.png)

Note that this does work better for many links when there is sufficient room: if we add in the tour link, and debug mode is enabled (otherwise those other two don't display), I don't know any way to avoid that hanging bullet on the first line:

![image](https://cloud.githubusercontent.com/assets/442115/10532046/927bf77c-7387-11e5-8539-a698990f8b0a.png)

Facebook's solution (from #205) has the same problem, and it wouldn't happen in debug mode, so this may not be an issue at all.

Alternate option for the separator would be the smaller dot `\000b7` instead of the bullet &bull;, which might actually be what FB uses:

![image](https://cloud.githubusercontent.com/assets/442115/10532121/4709ace8-7388-11e5-8af9-674c594973be.png)

kanitw commented:
LGTM.   Thanks for changing this!  :) 

Minor: I like the smaller dot better than bullet, but no strong preference.  Feel free to pick one and merge.

kadamwhite commented:
@kanitw Great! I switched to the dot (I liked it slightly more as well) and fixed the debug button, which I'd broken by accident in my first commit.

This should probably be made into another issue, but cc @vlandham in case this falls under the purview of #199 : the "About" button @vlandham added has no function, and does not appear to ever have had any. What's the plan there?

kanitw commented:
> the "About" button

We will add about dialog, showing brief description of the project and credit everyone who involves in this project.  (Bocoup, IDL, Tableau Research)
